miro_preprocessor/provider.tf

```diff
@@ -0,0 +1,8 @@
+# Specify the provider and access details
+provider "aws" {
+  region = "${var.aws_region}"
+
+  version = "0.1.4"
+}
+
+data "aws_caller_identity" "current" {}
```

Comment:
Do we need that? I think that was just needed for SNS topics in the shared infra pot.